tsinfer/formats.py

```diff
@@ -2351,7 +2356,10 @@ def __init__(
         sample_mask=None,
         site_mask=None,
         sites_time=None,
+        sequence_length=None,
     ):
+        self._sequence_length = sequence_length
+        self._contig_index = None
```

Comment:
Need to define ``self._contig_id`` here also, or will get attribute error when not defined.

Reply:
Fixed, and I've added checking this to the tests.